editor.js

```diff
@@ -3,7 +3,7 @@
 	Extension Editor for XKit 7
 	Version 1.0
 
-	(c) 2011 - 2013 STUDIOXENIX
+	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
```

Comment:
it actually should be "(c) 2015 multiple authors".

unlike studioxenix we're not one homogenous entity and can't really be
thought of as such from a copyright point of view

[IANAL, TINLA]

On Tue, Sep 1, 2015 at 3:46 PM Wolvan <notifications@github.com> wrote:

> In editor.js
> <https://github.com/new-xkit/XKit/pull/405#discussion_r38462712>:
>
> > @@ -3,7 +3,7 @@
> >  	Extension Editor for XKit 7
> >  	Version 1.0
> >
> > -	(c) 2011 - 2013 STUDIOXENIX
> > +	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
>
> OK, yeah. Point taken
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/405/files#r38462712>.
>